pkg/cache/resource.go

```diff
@@ -51,6 +54,7 @@ var (
 		RouteType,
 		ListenerType,
 		SecretType,
+		RuntimeType,
```

Comment:
Most of the changes to support RTDS just mimic what's been done for EDS, LDS, RDS, etc.